public/omni/cluster-management/scale-your-cluster/karpenter.mdx

```diff
@@ -115,7 +117,7 @@ name: <cluster-name> # Replace with your cluster name
 kubernetes:
   version: v1.34.1 # Replace this version with your preferred version of Kubernetes
 talos:
-  version: v1.11.3 # Replace this version with your preferred version of Kubernetes
+  version: ${release}.3 # Replace this version with your preferred version of Kubernetes
```

Comment:
I don't think I understand what't the goal here in `.3`  It says Kubernetes also which is wrong